ramls/loans-history-items.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Collection of loan history items",
+  "type": "object",
+  "properties": {
+    "loans-history": {
```

Comment:
I believe we usually use camel case for JSON property names. Is there a reason why this is hyphenated instead?

Reply:
No, changed.